app/controllers/works_controller.rb

```diff
@@ -0,0 +1,82 @@
+class WorksController < ApplicationController
+  def index
+    @albums = Work.albums
+    @books = Work.books
+    @movies = Work.movies
+  end
+
+  def show
+    id = params[:id]
+    @work = Work.find_by(id: id)
+  end
+
+  def new
+    @work = Work.new
+  end
+
+  def create
+    @work = Work.new(work_params) #instantiate a new work
+    if @work.save # save returns true if the database insert succeeds
+      success_message = "Successfully created #{@work.category} #{@work.id}"
+      flash_success(success_message)
+
+      redirect_to work_path(@work) # goes to work page
+      return
+    else # save failed
+      warning_message = "A problem occurred: Could not create work"
+      flash_warning(warning_message, @work)
+
+      render :new # show the new work form view again
```

Comment:
You need to include a status code. 
```suggestion
      render :new, status: :bad_request # show the new work form view again
```